src/linker/Linker.Steps/IPerAssemblyStep.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Mono.Cecil;
+
+namespace Mono.Linker.Steps
+{
+
+	public interface IPerAssemblyStep
```

Comment:
This needs much better name. Looking at the code this runs during Mark step only.

Reply:
Any suggestions? I tentatively changed it to `IMarkAssemblyStep`.

Reply:
With the registration-based model I instead called it `IMarkHandler`